tmt/schemas/common.yaml

```diff
@@ -424,6 +424,12 @@ definitions:
       name:
         type: string
 
+      url:
```

Comment:
If you do this, `prepare` should also be adapted. 👍 to make general to everywhere where it can.

Reply:
Yes, I have another PR for it: https://github.com/teemtee/tmt/pull/3862

Both are drafts because I am still ongoing :)

Reply:
Cool. I think it would be good to make it general (think it is already something in `tmt.utils`) and combine them together so that the schema tests don't fail

Reply:
@LecrisUT I am not sure if I get your point, you mean adding `url` and `ref` into prepare/finish `shell.yaml` instead of the common one?

Reply:
I think modifing the common schema would be preferred, but that means you'll have to adjust both implementations (unless they are all optional in which case it could be separate).

But I believe the implementation for both of them would be identical anyway so I don't think you need to separate them into different PRs. Up to your preference though.

Reply:
@happz so I should do the logic modification only in `prepare` plugin and then somehow reuse it on `finish` shell?